### PR TITLE
feat: add documentation for w3admin related capabilities

### PR DESCRIPTION
**Pull request by travis**
I think these deserve their own spec (and that we could potentially fold the rate-limits spec into this one) since they all act against the service resource itself.

### CODE REVIEW

File: w3-admin.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+}
+```
+
+#### errors
+
+`CustomerNotFound`
+
+#### capability definition
+
+```javascript=
```
Comment (alanshaw) [inline w3-admin.md:94]: Only supported in hackmd

```suggestion
```javascript
```

File: w3-admin.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+}
+```
+
+#### errors
+
+`SubscriptionNotFound`
+
+#### capability definition
+
+```javascript=
```
Comment (alanshaw) [inline w3-admin.md:134]:
```suggestion
```javascript
```

File: w3-admin.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+Get information about a customer.
+
+#### inputs
+
+`customer: DID<mailto>`
+
+#### returns
+
+```typescript
+  did: AccountDID
```
Comment (alanshaw) [inline w3-admin.md:83]:
```suggestion
```typescript
{
  did: AccountDID
```

File: w3-admin.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+}
+```
+
+#### errors
+
+`ConsumerNotFound`
+
+#### capability definition
+
+```javascript=
```
Comment (alanshaw) [inline w3-admin.md:55]:
```suggestion
```javascript
```

File: w3-admin.md [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+  }),
+  derives: (child, parent) => {
+    return (
+      and(equalWith(child, parent)) ||
+      and(equal(child.nb.consumer, parent.nb.consumer, 'consumer')) ||
+      ok({})
+    )
+  },
+})
+```
```
Comment (alanshaw) [inline w3-admin.md:70]: I think we need a better way to express this as this is very ucanto oriented and there are a bunch of unknown dependencies in this code snippet.

(not blocking for this PR...but maybe open an issue and we'll get round to addressing it when possible)

**Review by alanshaw — APPROVED** [review]
Please accept the typo suggestions 🙏

File: w3-admin.md (hunk 5 above)
Comment (travis) [reply w3-admin.md:70]: agreed - done! https://github.com/web3-storage/specs/issues/78

**Pull request merged**